docs/reference/run.md

```diff
@@ -773,7 +773,7 @@ capabilities using `--cap-add` and `--cap-drop`. By default, Docker has a defaul
 list of capabilities that are kept. The following table lists the Linux capability options which can be added or dropped.
 
 | Capability Key | Capability Description |
-| :----------------- | :---------------| :-------------------- |
+| -------------- | ---------------------- |
```

Comment:
Yeah, it fixes #14177. Which was not fixed properly earlier. You suggested the same change there.